packages/react-components/src/components/styles/UnsupportedEnvironment.styles.tsx

```diff
@@ -58,3 +69,18 @@ export const iconStyles: IIconStyles = {
     paddingBottom: '1rem'
   }
 };
+
+/**
+ * @internal
+ */
+export const continueAnywayButtonStyles = (theme: Theme): IButtonStyles => {
+  return {
+    root: {
+      backgroundColor: '#C4314B',
+      margin: '1rem',
```

Comment:
nit: consider using stack childrenTokens instead of margin and padding directly for a number of these styles